src/test/java/com/google/api/codegen/testdata/library_gapic.yaml

```diff
@@ -4,7 +4,7 @@ language_settings:
   java:
     package_name: com.google.gcloud.pubsub.spi
   python:
-    package_name: library
+    package_name: google.example.library.v1
```

Comment:
I am wondering what this means. Can you elaborate it?

Naming of 'google.example.library.v1' might be problematic, because it is same as the namespace for gRPC messages.  In many examples, it is not -- like `google.pubsub.v1` grpc vs `google.cloud.pubsub.v1` gapic, and I am still not sure where you intended to place the `enums.py` file.  Might be good to keep `library`, or use `google.cloud.example.library.v1`, so you can find potential filename bugs much earlier.